src/main/java/com/google/devtools/build/lib/bazel/rules/sh/ShBinary.java

```diff
@@ -63,8 +63,10 @@ public ConfiguredTarget create(RuleContext ruleContext)
     ruleContext.registerAction(
         new ExecutableSymlinkAction(ruleContext.getActionOwner(), src, symlink));
 
-    NestedSetBuilder<Artifact> filesToBuildBuilder =
-        NestedSetBuilder.<Artifact>stableOrder().add(src).add(symlink);
+    NestedSetBuilder<Artifact> filesToBuildBuilder = NestedSetBuilder.<Artifact>stableOrder()
+        .add(src)
+        .add(symlink)
+        .addAll(ruleContext.getPrerequisiteArtifacts("data", Mode.DONT_CHECK).list());
```

Comment:
It shouldn't be necessary to add data deps to files to build.

Reply:
Ah, got it. Changed the code to add the prerequisite artifacts to `runfilesBuilder` instead of `filesToBuild`.